packages/app-store/_utils/oauth/parseRefreshTokenResponse.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from "zod";
+
+import { APP_CREDENTIAL_SHARING_ENABLED } from "@calcom/lib/constants";
+
+const minimumTokenReponseSchema = z.object({
+  access_token: z.string(),
+  //   Assume that any property with a number is the expiry
+  [z.string().toString()]: z.number(),
+  //   Allow other properties in the token response
+  [z.string().optional().toString()]: z.unknown().optional(),
+});
```

Comment:
If we are being sent tokens then they should at least contain the access token and the expiry date. The expiry date could be named different things such as expiry_date, expires_in, etc